test/gst-va/encode/10bit/vp9.py

```diff
@@ -15,7 +15,7 @@ class VP9_10EncoderBaseTest(EncoderTest):
   def before(self):
     super().before()
     vars(self).update(
-      codec         = "vp9",
+      codec         = "vp9-10",
```

Comment:
The lib/gstreamer/va/encoder.py has some conditional checks for self.codec "vp9".  Need to update them to include "vp9-10" with this change.

Reply:
updated, thanks